android/app/src/main/java/app/organicmaps/bookmarks/data/Track.java

```diff
@@ -33,4 +33,9 @@ public class Track
   public long getTrackId() { return mTrackId; }
 
   public long getCategoryId() { return mCategoryId; }
+
+  public String getTrackDescription()
+  {
+    return BookmarkManager.INSTANCE.getTrackDescription(mTrackId);
```

Comment:
Descriptions might be bulky. The current implementation requires descriptions only when tracks are opened for editing. 

Fetching descriptions for each track and storing them locally may arise some performance issues. I am not very sure with analytics of performance issues but since in bookmarks also descriptions are directly loaded from core so i thought it might have designed in such a way for some reasons so i did that in same way : )